_docs-2-0/troubleshooting/peformance.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Performance
+category: troubleshooting
+order: 5
+---
+
+Accumulo can be tuned to improve read and write performance.
+
+## Read performance
+
+1. Enable [caching] on tables to reduce reads to disk.
+
+1. Enable [bloom filters][bloom-filters] on tables to limit the number of disk lookups.
+
+1. Decrease the [major compaction ratio][compaction] of a table to decrease the number of
+   files per tablet. Less files reduces the latency of reads.
+
+## Write performance
+
+1. Enable [native maps][native-maps] on tablet servers to prevent Java garbage collection pauses
+   which can slow ingest.
+
+1. [Pre-split new tables][split] to distribute writes across multiple tablet servers.
+
+1. Ingest data using [multiple clients][multi-client] or [bulk ingest][bulk] to increase ingest throughput.
+
+1. Increase the [major compaction ratio][compaction] of a table to limit the number of major compactions
+   which improves ingest peformance.
+
```

Comment:
Setting `table.file.compress.type=snappy` can increase write spead at the expense of using more disk space.